Add tips section for calendar components (#6354)
yamada-ui/yamada-ui: www/contents/components/(components)/calendar.mdx

<Calendar.Root />
```

:::tip

> If you want to use a date input field in a form, use [DatePicker](/docs/components/date-picker).

Reviewer: Let's keep this concise and avoid implying that `Calendar` should not be used for date selection.

```suggestion
If you want to have a date entered, use [DatePicker](/docs/components/date-picker).
```

> If you want to use it as an input field in a form, use [DatePicker](/docs/components/date-picker).

Reviewer 2: This reads a little awkwardly to me. I think it becomes clearer if the tip describes `DatePicker` more directly as the form-input option for date selection.

```suggestion
If you want to use a date input field in a form, use [DatePicker](/docs/components/date-picker).
```
